app/controllers/books.js

```diff
@@ -0,0 +1,4 @@
+import Controller from '@ember/controller';
```

Comment:
✂️

Reply:
Deleting it causes disappearing half of the content in the app.

Reply:
✂️, in that case, means - remove this file completely.